### DISCUSSION TITLE
Installation guide for OpenShift

### DISCUSSION THREAD

**Discussion by jax79sg** [triton-inference-server/server#5436] [Q&A] [answered]
Hi, can i ask if there's an installation guide or helm chart for installation of Triton with HPA on Openshift?

**Comment by the-david-oy** [comment, marked as answer]
There seems to be an NVIDIA lab on deploying an AI model that uses OpenShift with Triton that can be publicly viewed. You may be able to sign up for it for free as well to access the files, but it should be a great resource either way: https://docs.nvidia.com/launchpad/ai/classification-openshift/latest/index.html

This has some information about horizontal pod autoscaling, but in the context of Kubernetes: https://developer.nvidia.com/blog/deploying-nvidia-triton-at-scale-with-mig-and-kubernetes/

CC: @tanayvarshney
